config/plugins/visualizations/openseadragon/templates/openseadragon.mako

```diff
@@ -29,8 +29,9 @@
             const prefixUrl = "//openseadragon.github.io/openseadragon/images/";
             const simple_image_url = "${h.url_for(controller='/datasets', action='index')}/" + hda_id + "/display";
             const xmlns = "http://schemas.microsoft.com/deepzoom/2008";
+            const image_types = ["gif", "jpg", "png", "tiff"];
 
-            if (hda_ext == 'jpg' || hda_ext == 'png') {
+            if (image_types.indexOf(hda_ext) >= 0) {
```

Comment:
Maybe you can use image_types.includes(hda_ext) ?

Reply:
Done here https://github.com/galaxyproject/galaxy/pull/11017/commits/8e41f40041a53092a90428f40bf840c6fa94243f